modules/apps/object/object-web/src/main/java/com/liferay/object/web/internal/object/definitions/portlet/action/GetObjectRelationshipInfoMVCResourceCommand.java

```diff
@@ -54,6 +54,10 @@ protected void doServeResource(
 				"objectRelationshipTypes",
 				ObjectRelationshipUtil.getObjectRelationshipTypes(
 					objectDefinition, _systemObjectDefinitionManagerRegistry)
+			).put(
+				"parameterEndpoint",
```

Comment:
@murilostodolni can we rename it to `restContextPath`?

Reply:
Yes, I agree! But in` edit_object_relationship.jsp` we call `parameterEndpoint`. Do we refactor there as well?